### ISSUE TITLE
Internal server error when trying to download CSV of all products in import products page

### ISSUE THREAD

**Issue by WebFreak001** [openboxes/openboxes#1382]
1. Click the "Import Products" button in products
2. Press "Download CSV of all products"

Application Version:  0.8.10

I currently have 2 products in my database.

```
groovy.lang.MissingMethodException: No signature of method: org.pih.warehouse.product.ProductService.exportProducts() is applicable for argument types: (java.util.ArrayList, null) values: [[<item name 1>, <item name 2>], null]
Possible solutions: exportProducts(), exportProducts(java.util.List), exportProducts(java.util.List, boolean), importProducts(java.lang.Object), importProducts(java.lang.Object, java.lang.Object)

	at org.pih.warehouse.product.ProductController$_closure29.doCall(ProductController.groovy:798)

	at org.pih.warehouse.product.ProductController$_closure29.doCall(ProductController.groovy)

	at grails.plugin.springcache.web.GrailsFragmentCachingFilter.doFilter(GrailsFragmentCachingFilter.groovy:66)

	at net.sf.ehcache.constructs.web.filter.Filter.doFilter(Filter.java:86)

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622)

	at java.lang.Thread.run(Thread.java:748)
```

**Comment by jmiranda** [comment]
This is a known bug that is fixed in 0.8.10.the workaround is to click the Export product attributes link to the right. It is functionally equivalent as long as you have not created any custom attributes.

**Comment by jmiranda** [comment]
Fixed in 0.8.11
